arelle/plugin/validate/DBA/rules/fr.py

```diff
@@ -301,6 +301,45 @@ def rule_fr56(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_fr63(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.FR63: Items in the balance sheet must be less than or equal to the balance sheet total. Assets must be
+    less than or equal to the balance sheet total. Applies to both the year's figures and comparative figures.
+    """
+    modelXbrl = val.modelXbrl
+    asset_facts = modelXbrl.factsByQname.get(pluginData.assetsQn)
+    if asset_facts is not None:
+        for asset_fact in asset_facts:
+            if asset_fact.xValid >= VALID:
+                concepts_in_error = []
+                facts_in_error = []
+                for balance_sheet_qn in pluginData.balanceSheetQnLessThanOrEqualToAssets:
+                    balance_sheet_qn_facts = modelXbrl.factsByQname.get(balance_sheet_qn)
+                    if balance_sheet_qn_facts is not None:
+                        for balance_sheet_qn_fact in balance_sheet_qn_facts:
+                            if (balance_sheet_qn_fact.xValid >= VALID and
+                                    asset_fact.contextID == balance_sheet_qn_fact.contextID and
```

Comment:
Out of curiosity, does it need to apply to facts with the same period but different dimensions?  (I know nothing about this tagging, so maybe dimensions are never applied.)

Reply:
Based on several filed danish reports that I have looked at, the contexts including dimensionality all seem to be the same across these concepts. If I focus just on the instant, instead of the context, all the reports I looked at would fire FR63 incorrectly.